packages/foreman/nodejs-patternfly-react/nodejs-patternfly-react.spec

```diff
@@ -1,10 +1,11 @@
 %global npm_name patternfly-react
+%global enable_tests 1
```

Comment:
We dropped this from the latest template. I've released 4.0.0 with it and should also be compatible with the latest nodejs. Do note that you may need to rebase your branch to include f4cb5706cb8838e73b8eb1d7b8b071e568b5c61e which makes it compatible.